lms/static/js/student_account/models/RegisterModel.js

```diff
@@ -1,61 +1,57 @@
-var edx = edx || {};
-
-(function($, Backbone) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'backbone', 'jquery.url'], function($, Backbone) {
+
+        var RegisterModel = Backbone.Model.extend({
```

Comment:
Instead of assigning the model to a variable and returning that you can just return the model, i.e.
```
return Backbone.Model.extend({
```